otter/test/test_scheduler.py

```diff
@@ -79,31 +94,32 @@ def _start_service(self):
         self.scheduler_service.running = True
 
     def test_start_service(self):
-        """
-        startService() calls super's startService() and creates SetPartitioner object
-        """
```

Comment:
Yeah, I'll just discard the hunks. First, to find an efficient way to revert the hunks in this one...